spaces/views.py

```diff
@@ -0,0 +1,47 @@
+import json
+
+from django.views       import View
+from django.http        import JsonResponse
+from django.db.models   import Sum
+
+from spaces.models      import (Space, Location, Image, Bedroom, BedroomBed, 
+                                Option, SpaceOption, Tag, SpaceTag, PlaceType)
+
+
+class SpaceListView(View):
+    def get(self, request):
+        place_type = request.GET.get("type")
+        if place_type is None:
+            spaces = Space.objects.all()
+        else:
+            spaces = Space.objects.filter(place_type__name=request.GET.get("type"))
```

Comment:
아래와 같은 방법으로 dictionary를 활용해서 filter를 구현해보세요.
이 방법으로 구현하기 위해서는 `unpacking` 개념을 공부해보시면 좋겠습니다.
```suggestion
        place_type = request.GET.get("type")

        filter_set = {}
        if place_type:
            filter_set['play_type__name'] = place_type
        spaces = Space.objects.filter(**filter_set)
```

Reply:
@lordmyshepherd  프로젝트 끝나면 unpacking  자세히 공부해봐야겠어요,,,!